corehq/ex-submodules/phonelog/utils.py

```diff
@@ -108,7 +108,9 @@ def _process_user_error_subreport(domain, xform):
             xform_id=xform.form_id,
             i=i,
             app_id=error['app_id'],
-            version_number=int(error['version']),
+            # beta versions have 'version', but the name should now be
+            # 'app_build'.  Probably fine to remove after June 2016.
```

Comment:
if you use an old version of commcare it'll still send `version` no? won't June 2016 deadline mean that everyone that uses commcare updated their version to the latest?